First commit-development complete

Tejashreenk opened the pull request.

bennytheshap committed:
Initial commit

tejashreeStrative committed:
First commit-development complete

bennytheshap commented on flutter_tests/accessibility_contrast_2_test.dart:
Why is this test in the repo?
This repo should be as minimal as possible. 
We'll then make another repo on gitlab that forks this one and adds tests that we use for CSE 340. See plan here: https://mattermost.cs.washington.edu/cse340-24fall-instructionalteam/pl/eraewb1tftdcjfdyt7bkjz59fy

bennytheshap replied:
Thanks for explaining. I think it'd be great to provide examples in a docs or examples directory, but let's not use examples that are actual undisclosed tests we will use in class.

bennytheshap commented on flutter_tests/accessibility_contrast_and_spacing_test.dart:
I suggest we combine this file with `flutter_tests/accessibility_contrast_2_test.dart` when we create the downstream 340 base test repo

bennytheshap commented:
There are a bunch of files and directories in these commits that seem like they should not be here. 
Examples include:

1. `test_results.json` (I suggest adding to the `.gitignore`)
2.  `test_suite`  (and everything within)
3.  `test_data` (and everything within)
4. `to_zip` (and everything within)

Unless I'm mistaken, 2 and 3 are the python code that we are eliminating our dependency on.

Please delete all of these (and, if possible, [remove the tests from the repo history](https://stackoverflow.com/questions/43762338/how-to-remove-file-from-git-history)).

bennytheshap commented:
> _ No description provided. _

Please include descriptions with all future PRs

bennytheshap reviewed (changes requested):
Please address the comments I have left on this PR. Let me know when you have, and I will be happy to re-review.

tejashreeStrative committed:
added content to readme.md, removed the test_suite folder, added tree structure print statements

tejashreeStrative committed:
unicode added in print statements, content added to readme file

bennytheshap committed:
Reorganized repo by:
- Removing build/test/zip artifacts
- Moving setup and autograder run scripts to root
- Moving example config and output into a new docs directory (and updating README accordingly)

bennytheshap committed:
fixed path to test_results.json in gitignore
